.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Deploy to DockerHub
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types: # This configuration does not affect the page_build event above
+      - created
+
+
+# Environment variables available to all jobs and steps in this workflow
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_ORGANIZATION: ${{ secrets.DOCKER_ORGANIZATION }} 
+  IMAGE: wisp-users
+
+jobs:
+  setup-build-publish-deploy:
+    name: Setup, Build, Publish, and Deploy
+    runs-on: ubuntu-latest
+    steps:
+
+    # Checkout Commit
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    # Build the Docker image
+    - name: Build
+      run: |        
+        docker build -t "$DOCKER_ORGANIZATION"/"$IMAGE":"$GITHUB_SHA" \
+          --build-arg GITHUB_SHA="$GITHUB_SHA" \
+          --build-arg GITHUB_REF="$GITHUB_REF" .
+        docker tag "$DOCKER_ORGANIZATION"/"$IMAGE":"$GITHUB_SHA" "$DOCKER_ORGANIZATION"/"$IMAGE":latest
```

Comment:
Should have a separate step for tagging (in preparation for when we move to charts since we'll have more things to tag)